tests/integration/test_charms.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2022 Canonical Ltd.
```

Comment:
What would be the reason you created an `integration` directory?

Reply:
assuming there would be another directory for unit tests